src/OpenTelemetry.Instrumentation.AspNetCore/AspNetCoreTraceInstrumentationOptions.cs

```diff
@@ -93,6 +93,11 @@ internal AspNetCoreTraceInstrumentationOptions(IConfiguration configuration)
     /// </remarks>
     public bool RecordException { get; set; }
 
+    /// <summary>
+    /// Gets or sets a value indicating whether SignalR activities are recorded.
+    /// </summary>
+    public bool DisableAspNetCoreSignalRSupport { get; set; }
```

Comment:
Naming suggestion:
```suggestion
    public bool EnableAspNetCoreSignalRSupport { get; set; }
```
A couple thoughts in there:
1. Naming the property Enable rather than Disable avoids reasoning with double-negatives when the value is false.
2. It matches the pattern for EnableGrpcAspNetCoreSupport below.